codersdk/deployment.go

```diff
@@ -1502,10 +1508,11 @@ when required by your organization's security policy.`,
 			// The default value is essentially "forever", so just use 100 years.
 			// We have to add in the 25 leap days for the frontend to show the
 			// "100 years" correctly.
-			Default: ((100 * 365 * time.Hour * 24) + (25 * time.Hour * 24)).String(),
-			Value:   &c.MaxTokenLifetime,
-			Group:   &deploymentGroupNetworkingHTTP,
-			YAML:    "maxTokenLifetime",
+			Default:     ((100 * 365 * time.Hour * 24) + (25 * time.Hour * 24)).String(),
```

Comment:
Note: this value currently does not display correctly (screenshot from dev.coder.com):

![image](https://github.com/coder/coder/assets/4949514/f300a701-a8a7-4138-90c7-da89a9f4eaf1)

We can modify this in a separate PR but I would consider it out of scope here.